Coding/server/data/index.html

```diff
@@ -1 +1,17 @@
-<!DOCTYPE html><html lang=""><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="icon" href="/favicon.ico"><title>complatform</title><link href="/js/about.125f2d3c.js" rel="prefetch"><link href="/css/app.42682c5f.css" rel="preload" as="style"><link href="/js/app.6bc2d095.js" rel="preload" as="script"><link href="/js/chunk-vendors.d69ca448.js" rel="preload" as="script"><link href="/css/app.42682c5f.css" rel="stylesheet"></head><body><noscript><strong>We're sorry but complatform doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div><script src="/js/chunk-vendors.d69ca448.js"></script><script src="/js/app.6bc2d095.js"></script></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="/favicon.ico">
+    <title>complatform</title>
+  <link href="/js/about.js" rel="prefetch"><link href="/js/app.js" rel="preload" as="script"><link href="/js/chunk-vendors.js" rel="preload" as="script"></head>
+  <body>
+    <noscript>
+      <strong>We're sorry but complatform doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
+    </noscript>
+    <div id="app"></div>
+    <!-- built files will be auto injected -->
+  <script type="text/javascript" src="/js/chunk-vendors.js"></script><script type="text/javascript" src="/js/app.js"></script></body>
```

Comment:
Move ```</body>``` to next line and consider idention.

Reply:
Fixed with a26d081